Makefile

```diff
@@ -8,8 +13,12 @@ all: dockerrun
 	make dockerbuild
 
 dockerbuild:
+	cp ./docker/config/webapp.env.template ./docker/config/webapp.env
+	sed -i 's/#SITE_URL#/$(subst /,\/,${SITE_URL})/g' ./docker/config/webapp.env
```

Comment:
This breaks on OS X. It works this way:
```
sed -i '' -e 's/#SITE_URL#/$(subst /,\/,${SITE_URL})/g' ./docker/config/webapp.env
```
Although it might break on other platforms...

Reply:
I was running that command under linux envs, that's probably it.

Reply:
Also, that maybe good to check if that will work on Windows 10 with bash subsystem if someone uses Windows for development.